### PR TITLE
Update HBaseMutationCoder Java Doc

### PR DESCRIPTION
**Pull request by kevinsi4508**

### CODE REVIEW

**Comment by kevinsi4508** [comment]
I signed it!

**Comment by kevinsi4508** [comment]
I signed it!

**Comment by sduskis** [comment]
LGTM

**Comment by kevinsi4508** [comment]
Jeff, second look?

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HBaseMutationCoder.java [hunk 1]
```diff
@@ -27,14 +27,16 @@
...
 import com.google.cloud.dataflow.sdk.Pipeline;
 import com.google.cloud.dataflow.sdk.coders.AtomicCoder;
 import com.google.cloud.dataflow.sdk.coders.Coder;
 import com.google.cloud.dataflow.sdk.coders.CoderException;
 
 /**
- * A {@link Coder} that serializes {@link Mutation} objects used by the Cloud Bigtable {@link Sink}.
+ * When Dataflow notices a slowdown in executing Puts and Deletes, it will send those Puts and
+ * Deletes to be processed on another Dataflow machine. This class handles the serialization and
+ * desrialization of the Puts and Deletes.
```
Comment (hegemonic) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HBaseMutationCoder.java]: "deserialization" is misspelled

**Comment by hegemonic** [comment]
One small comment, but otherwise LGTM.

**Comment by kevinsi4508** [comment]
thanks!

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HBaseMutationCoder.java (hunk 1 above)
Comment (kevinsi4508) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HBaseMutationCoder.java]: done.

**Pull request merged**